src/hotspot/share/gc/shenandoah/shenandoahConcurrentGC.cpp

```diff
@@ -72,6 +72,21 @@ class ShenandoahBreakpointMarkScope : public StackObj {
   }
 };
 
+class ShenandoahGlobalCoalesceAndFill : public ShenandoahHeapRegionClosure {
```

Comment:
Not sure how this ended up `shenandoahConcurrentGC.cpp`, last I checked it was defined in `shenandoahHeap.cpp`.

Reply:
Not sure either.  It is in both places in my branch.  I see no reason for it to be here so am removing it.